src/components/code-editor/index.jsx

```diff
@@ -1,10 +1,12 @@
-import { h, Component, render } from 'preact';
+import { Component, render } from 'preact';
 import codemirror from 'codemirror';
 import cx from '../../lib/cx';
 import 'codemirror/mode/jsx/jsx';
 import 'codemirror/addon/comment/comment';
 import 'codemirror/lib/codemirror.css';
 import style from './style.module.css';
+// TODO: Fix me, Vite doesn't seem to like :global in CSS Modules
+import './temp.css';
```

Comment:
Reminder for myself

Reply:
More specifically, Vite's CSS Modules processor (whatever it is) dislikes `:global {}` blocks.

This is fine, larger blocks we can just extract out to separate plain `.css` files.